dev-container-feature/src/dapr-cli/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -e
+
+echo "Activating feature 'dapr-cli'"
+
+get_latest_release() {
+  curl --silent "https://api.github.com/repos/dapr/cli/releases/latest" |
+  grep '"tag_name":' | sed -E "s/.*\"v([^\"]+)\".*/\1/"
+}
+
+VERSION=${VERSION:-"latest"}
+
+if [ "${VERSION}" = "latest" ]; then
+  VERSION=$(get_latest_release)
+fi
+
+curl -SsL https://github.com/dapr/cli/releases/download/v"${VERSION}"/dapr_linux_amd64.tar.gz | \
```

Comment:
Please make sure to detect the system's architecture, as we want the feature to be usable on ARM64 (and maybe ARM 32-bit) too

Reply:
Good point - added :-)